tigerpath/templates/tigerpath/about.html

```diff
@@ -35,118 +35,34 @@ <h1 class="mb-3">About TigerPath</h1>
-            <li><p>Let the user customize requirements that they have fulfilled through other means (such as study abroad)</p></li>
-            <li><p>Inform the user of when courses are typically offered (such as how frequently the course is offered and which semester it is typically offered in)</p></li>
-            <li><p>Start putting in requirements for other majors</p></li>
-          </ul>
-
-          <p class="tl-section">April 27th: Alpha test</p>
-          <ul>
-            <li><p>By this time, we should have full basic functionality of the app, including the following:</p></li>
-            <li><p>Being able to retrieve saved user data during a later session</p></li>
-            <li><p>Having a presentable UI</p></li>
-            <li><p>Requirements customization and course time predictions</p></li>
-            <li><p>Continue propagating app for people to try out</p></li>
-          </ul>
-
-          <p class="tl-section">April 29th:</p>
-          <ul>
-            <li><p>Continue putting in requirements for other majors</p></li>
-            <li><p>Work on other additional features</p></li>
-            <li><p>Start working on presentation/final documentation</p></li>
-          </ul>
-
-          <p class="tl-section">May 3rd: Beta Test</p>
-          <ul>
-            <li><p>The app is fully usable and useful to a user that would want to use it</p></li>
-            <li><p>Do a beta test release to the Princeton community</p></li>
-          </ul>
+    <section class="container mb-4">
+      <div class="text-section">
+        <h4>Description</h4>
+        <p>TigerPath is a web application that helps Princeton University students plan out their four-year course schedules.</p>
+        <p>There is currently no good way for students to manage what courses they’re taking and when they're taking them beyond the current or upcoming semester. Many students try to manually do this by using an Excel spreadsheet or pen-and-paper, but any mistake can easily lead to an overloaded semester or a need to take summer classes in order to fulfill missed requirements.</p>
+        <p>TigerPath aims to solve this problem by providing a platform for students to manage what courses they're taking during each semester and giving them feedback on the major requirements their schedule fulfills and does not fulfill. Our goal is for TigerPath to become a useful, enduring tool that ensures Princeton students are able to stay on top of requirements and graduate on time.</p>
+        <p>We currently fully support COS (AB and BSE), ELE, CBE, and Woody Woo, but we are continually adding more majors. If you'd like to be notified when your major is added to TigerPath, please fill out <a href="https://goo.gl/forms/pKxjmubIOSCOeR8L2">this form</a>!</p>
+      </div>
 
-          <p class="tl-section">May 9th, 10th: Demo Days</p>
-          <ul><li><p>Finish presentation</p></li></ul>
+      <div class="text-section">
```

Comment:
Same thing with other headers. Not enough spacing before the headers. They seem to be more attached to the section before them than the one after them.

![image](https://user-images.githubusercontent.com/9757500/40261153-0fa28886-5acd-11e8-9ba1-c58efeef2b44.png)

Reply:
Not sure why we're getting different spacing.
<img width="1140" alt="image 2018-05-18 at 7 59 48 pm" src="https://user-images.githubusercontent.com/4218237/40262543-24fd232c-5ad6-11e8-943e-b01e771dc82e.png">

Reply:
Try now and see if it's fixed

Reply:
No good. Still there.

Reply:
It looks fine on the Privacy Policy page, so the formatting should be the same.
It seems that there, you have both `mb-4` and `mt-4`:
`<section class="container mb-4 mt-4">`
whereas in the about page, you only have `mb-4`
`<section class="container mb-4">`

Reply:
As I mentioned above, the Privacy Policy page looks right, so maybe it's some of the differences between them that cause this.